.github/workflows/deploy.yml

```diff
@@ -14,7 +14,7 @@ on:
 
 jobs:
   deploy:
-    if: github.event_name == 'release' || github.ref == 'refs/heads/main' || contains(github.event.head_commit.message, '[stagingdeploy]') || contains(github.event.head_commit.message, '[testnetdeploy]') || contains(github.event.head_commit.message, '[mainnetdeploy]')
+    if: contains(github.event.head_commit.message, '[stagingdeploy]') || contains(github.event.head_commit.message, '[testnetdeploy]') || contains(github.event.head_commit.message, '[mainnetdeploy]')
```

Comment:
I disabled this until we confirm our release strategy [in our doc](https://www.notion.so/textile/Validator-Releases-Playbook-84f90706d20f487aa3dbbbda3d59d563).

Probably what we'll confirm is that we should only delete `github.ref == 'refs/heads/main' `.